<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SnellsLawCalculatorClass</class>
 <widget class="QMainWindow" name="SnellsLawCalculatorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>689</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SnellsLawCalculator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>871</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QRadioButton" name="incidenceIndexRadioButton">
           <property name="text">
            <string>Refractive index 1 (n₁) </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="refractiveIndexRadioButton">
           <property name="text">
            <string>Refractive index 2 (n₂)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="angleOfIncidenceRadioButton">
           <property name="text">
            <string>Angle of incidence (θ₁)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="angleOfRefractionRadioButton">
           <property name="text">
            <string>Angle of refraction (θ₂)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="refractiveIndexLabel1">
         <property name="text">
          <string>Refractive index 1 (n₁) </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="incidenceIndexDoubleSpinBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="refractiveIndexLabel2">
         <property name="text">
          <string>Refractive index 2 (n₂)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="refractiveIndexDoubleSpinBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="angleOfIncidenceLabel">
         <property name="text">
          <string>Angle of incidence (θ₁)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="angleOfIncidenceDoubleSpinBox"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>deg</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="angleOfRefraction">
         <property name="text">
          <string>Angle of refraction (θ₂)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="angleOfRefractionDoubleSpinBox"/>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>deg</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>671</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/SnellsLawCalculator/img.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>894</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="SnellsLawCalculator.qrc"/>
 </resources>
 <connections/>
</ui>
